embark/users/views.py

```diff
@@ -90,3 +99,71 @@ def signout(request):
     logger.debug("Logout user %s", request.user)
     logout(request)
     return render(request, 'uploader/login.html', {'success_message': True, 'message': 'Logout successful.'})
+
+
+@csrf_exempt
+@login_required(login_url='/' + settings.LOGIN_URL)
+@require_http_methods(["GET", "POST"])
+def password_change(request):
+    if request.method == "POST":
+        logger.debug(request.POST)
+        user = get_user(request)
+
+        data = {k: v[0] for k, v in dict(request.POST).items()}
+        logger.debug(data)
+        try:
+            body = {k: v[0] for k, v in dict(request.POST).items()}
+            try:
+                oldPassword = body['oldPassword']
```

Comment:
![image](https://user-images.githubusercontent.com/62940240/149895610-fe4a4a72-0393-4aa3-ad15-745db382dbc6.png)

vars should have a underscore in between words "new_password"
*only affects python not the js or html so the assignment value can stay the same*